pkg/reconciler/clusterducktype/clusterducktype.go

```diff
@@ -18,11 +18,13 @@ package clusterducktype
 
 import (
 	"context"
-	"sort"
```

Comment:
UTs for this coming later?

Reply:
yeah, I am still working out the logic of the reconciler and the shape of the object. Will make a real set of integration tests soon as this is more stable

Reply:
ack